networking/external_dns_operator/nw-control-dns-records-public-hosted-zone-gcp.adoc

```diff
@@ -0,0 +1,142 @@
+[id="nw-control-dns-records-public-hosted-zone-gcp_{context}"]
+= Controlling DNS records on an public hosted zone for GCP by using Red Hat External DNS Operator 
+
+.Procedure
+
+. Export the kubeconfig file.
++
+[source,terminal]
+----
+$ export KUBECONFIG=~/<path to>/kubeconfig
+----
+
+. Check the user. The user must have access to the `kube-system` namespace. If you don’t have the credentials and want to fetch from the `kube-system` namespace then use the client.
```

Comment:
Check the user. The user must have access to the `kube-system` namespace if you don’t have the credentials as you can fetch the credentials from the `kube-system` namespace to use the cloud provider client. Eg- aws.